presentation/chapters/es-ES/unsafe.chapter

```diff
@@ -0,0 +1,65 @@
+## Unsafe Rust
+
+El sistema de tipo de Rust provee muchas garantías, pero en ocasiones esto conlleva que soluciones específicas sean difíciles o imposibles.
+
+Por esta razón, Rust tiene el concepto de "código inseguro".
+
+---
+
+El código inseguro tiene permitido:
+-   acceder memoria libremente
+-   utilizar apuntadores crudos (raw pointers)
```

Comment:
I think we use more the term `punteros` than `apuntadores` see https://es.wikipedia.org/wiki/Puntero_(inform%C3%A1tica)